src/components/Header/Header.js

```diff
@@ -15,6 +16,9 @@ const Header = ({ ChangePage, CurrentPage, Resume }) => {
           />
         ))}
       </ul>
+      <Link to="/login" className="btn btn-primary btn-sm col-1">
+          Login
```

Comment:
Prettier 2 space indentation, please?

Reply:
Sorry I didn't git this one @praveenscience